apps/qubit/modules/digitalobject/actions/metadataComponent.class.php

```diff
@@ -61,10 +61,15 @@ public function execute($request)
     $this->referenceCopyDenyReason = false;
     $this->thumbnailCopyDenyReason = false;
 
-    // Group permissions always control access to AtoM's representations if the digital object is related to an actor
-    if ($this->relatedToActor && QubitAcl::check($this->resource->object, 'read'))
```

Comment:
Maybe I misunderstood how permissions work, but if we remove the `QubitAcl::check` doesn't that mean we can't limit access anymore? I think I picked up that logic for actors from the `Must have read privileges to view` comment (and related changes) in https://github.com/artefactual/atom/commit/77f91b2909f276950ae295b55cbb8fdce589dbdc

Reply:
After carefully considering and reviewing the digital object permissions, I have come to the conclusion I don't want to touch them with a 10 foot pole. 😆   

I'll remove this change from the PR, and resubmit. 😅